.github/Stock Exchange.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="stayle.css">
+    <title>Stock Exchange</title>
+    <link rel="shortcut icon" href="dollar-coin-2149.png" type="image/x-icon">
+</head>
+
+<body>
+    <div class="marquee">
+        <!-- <ul class="marqueeData">
+        </ul> -->
+
+    </div>
+    <div class="container">
+        <h1>Search Nasdaq Stocks</h1>
+        <div class="search">
+            <input class="searchInput" type="search" id="search" name="search">
```

Comment:
You had to create the search form (text input and button) from the class, in JS, not HTML